LayoutTests/crypto/subtle/ecdh-derive-bits-length-limits.html

```diff
@@ -85,7 +85,7 @@
     return Promise.resolve().then(function(result) {
         // P-256
         return Promise.all([
-            deriveBits(P256, 0, 256),
```

Comment:
Nit: While you are at it, how about add:
`deriveBits(P256, 0)` also and such to all the cases so that they also get tested here. 
I am aware that such tests are in wpt, but this will read better in itself.

Reply:
Makes sense. As a matter of fact I've been thinking on porting those tests to WPT; anyway, this will be a follow up patch, so I'll add the new test cases here meanwhile.

Reply:
Now that I think about it, it's unfortunate the deriveBits utility function defines the "expectedLength"  as the last parameter. This makes more complex to add test cases for the "omitted" case. 

Do you think it's worth to do a refactoring of the test because of this ?